Casks/lg-onscreen-control.rb

```diff
@@ -1,6 +1,6 @@
 cask "lg-onscreen-control" do
-  version "5.47,dpRrWj2RVwyi5XuIZw6hg"
-  sha256 "9d006c3c704c6d3d9602a559e33749f5747377972e19f2c4182255d35814c68a"
+  version "5.48"
```

Comment:
Looks like it needs to be this

```suggestion
  version "5.48,uYXSwyUCNFBbSch9PFw"
```